pkg/plugins/models.go

```diff
@@ -97,8 +97,8 @@ type Includes struct {
 }
 
 func (e Includes) DashboardURLPath() string {
-	if e.Type != "dashboard" || len(e.UID) == 0 {
-		return ""
+	if len(e.UID) == 0 {
+		return "/d/" + e.Slug
```

Comment:
don't quite understand what this does - is it just a fallback if there's no UID? 🤔

Reply:
@ashharrison90 it was to make it so that an old app plugin I tried that had dashboards without uid showed up in the nav. Need to revert this